### PR TITLE
AppBar fixes

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
A few small fixes after the release.

### CODE REVIEW

File: src/elements/index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+export AppBar from './app-bar'
+export * from './app-bar'
 export Heading from './heading'
 export Button from './button'
 export * from './forms'
 export Label from './label'
+export Table from './tables'
```
Comment (Joozty) [inline src/elements/index.js]: Why did you change it? It works just fine for me in https://github.com/oacore/dashboard/pull/20

Comment (Joozty) [reply src/elements/index.js]: You export it two times, don't you?

Comment (viktor-yakubiv) [reply src/elements/index.js]: It's according to the spec. If I export *, it does not include default. So I do 2 exports, one for default and one for the rest

Comment (Joozty) [reply src/elements/index.js]: Anyway it's exported here so it doesn't really matter https://github.com/oacore/design/blob/master/src/elements/tables/index.js#L16

Comment (viktor-yakubiv) [reply src/elements/index.js]: Read carefully
https://github.com/oacore/design/pull/59/files#diff-a0a697338eb47b168a0854933a67eaa4L16

Comment (Joozty) [reply src/elements/index.js]: Sorry, my bad 😴
